feat: Product Localization Playbook blog post (#446)
tolgee/documentation: blog/2023-07-27-product-localization.md

The timeline is equally crucial, as it sets the pace for your product’s entry into new markets. Determine realistic deadlines for each stage of the localization process, from translation and quality assurance to implementation and testing. **Rushing the process can lead to errors and compromised quality**, while too much time can result in missed market opportunities.

Remember, **effective localization takes time and effort**, but with careful planning and resource allocation, you can ensure a successful launch of your product in new markets.

### Metrics and KPIs

Metrics and KPIs in localization are essential because they help you measure the success of your localization process. On the other hand, **measuring the success** of your localization process could be **challenging as localization doesn’t generate revenue directly** but indirectly through increased sales and customer satisfaction. It’s always best to use those KPIs that make sense for your project. Consider these examples of localization metrics and KPIs:

> - **Linguistic quality** metrics: These metrics measure the quality of your translations. You can use the [TAUS’ DQF-MQM](https://www.taus.net/data-services/dqf-mqm-error-annotation) – the industry standard for linguistic quality evaluation.

Reviewer: ```suggestion
- **Linguistic quality** metrics: These metrics measure the quality of your translations. You can use the [DQF-MQM error typology](https://www.taus.net/data-services/dqf-mqm-error-annotation) – the industry standard for linguistic quality evaluation, developed by TAUS.

```

- **Business** metrics: You can also use business metrics to measure the success of your localization process. These can be the target market share, customer retention rate, active monthly users, etc.
- **Cost** metrics: The metrics can give you valuable insight into how much money you spend on localization. You can use, for example, the cost per word to measure the cost of your localization process.
- **Website** metrics: Website metrics are a great way to see how your localized website is performing. You can use, for example, the bounce rate to measure how many visitors leave your website without taking any action and compare this number with the bounce rate of your original website.

### Roles and responsibilities

Roles and responsibilities **can make or break your localization** process. You should assign roles and responsibilities to your localization team and ensure everyone knows their responsibility. In an ideal world, you should have a **localization manager** responsible for the localization process and a team of localization and language specialists responsible for the localization itself.

> The responsibilities of the localization team could range from **localizing content**, through **evaluating localization quality**, to **developing localization workflows**. The responsibilities and size of the team depend on the complexity of your product, the target market, whether and how you want to [measure the success of the localization](#metrics-and-kpis), your budget and timeline, and other factors.

Reviewer: ```suggestion
The responsibilities of the localization team could range from **localizing content**, through **evaluating localization quality**, to **developing localization workflows**. The responsibilities and size of the team depend on the complexity of your product, the target market, whether and how you want to [measure the success of the localization](#metrics-and-kpis), your budget and timeline, as well as other factors.

```

Likewise, the **processes** you implement should complement your localization strategy. When incorporating MT, consider creating a process for post-editing the machine-translated content within the TMS to ensure optimal results. If you’re using a TMS, you can also create a process for managing the translation workflow, from translation to reviewing and approving the translated content.

## Localization strategy examples

### McDonald’s

Even though McDonald’s is a global brand and is considered a globalization behemoth, they still localize their products to fit the local market. In India, McDonald’s replaced the two traditional beef patties in a Big Mac with chicken ones to create the [Maharaja Mac](https://mcdonaldsblog.in/2016/11/chicken-maharaja-mac-the-indian-answer-to-big-mac/). The reason for this change is that in India, cows are considered sacred animals, and in most states of India, there is a law prohibiting beef consumption.

> In the Philippines, McDonald’s has introduced [McSpaghetti](https://www.mcdonalds.com.ph/our-food/chicken-platters) to their menu. The dish combines spaghetti noodles, tomato sauce, and grated cheese. This change is because spaghetti is a popular dish in the Philippines and is often served during birthday parties.

Reviewer: ```suggestion
In the Philippines, McDonald’s has introduced [McSpaghetti](https://www.mcdonalds.com.ph/our-food/chicken-platters) to their menu. The dish combines spaghetti noodles, tomato sauce and grated cheese. This change is due to the fact that spaghetti is a popular dish in the Philippines and is often served during birthday parties.

```

### Uber

Uber is an excellent example of a company offering genuinely local services. In 2015, Uber launched [UberAuto](https://www.uber.com/in/en/ride/uber-auto), which allows users to book an auto-rickshaw in India. As cash is a common payment method in India, Uber has also decided to allow users to pay for their rides in cash.

> ### Slack

Reviewer 2: I'd add a picture (with a source) from this article with the candy gift

> Slack has localized its product into more than 10 languages while maintaining a consistent brand identity and tone of voice. In their blog post, [How we localize Slack](https://slack.com/blog/collaboration/how-we-localize-slack), they show examples of how they localized (not just translated) messages and icons in their product. For example, the message displayed when you’ve read everything in your “Unreads” section, ”You’re all caught up. Here’s a pony.”, does not make sense in Spanish. So in Spanish, they localized it as “Has leído todo. Aquí tienes un caramelo de regalo.”, meaning “You’ve read everything. Here’s a candy gift.” – the icon was also changed to a candy.

Reviewer: ```suggestion
Slack has localized its product into more than 10 languages while maintaining a consistent brand identity and tone of voice. In their blog post, [How we localize Slack](https://slack.com/blog/collaboration/how-we-localize-slack), they show examples of how they localized (not just translated) messages and icons in their product. For example, the message displayed when you’ve read everything in your “Unreads” section, ”You’re all caught up. Here’s a pony”, does not make sense in Spanish. So in Spanish, they localized it as “Has leído todo. Aquí tienes un caramelo de regalo”, meaning “You’ve read everything. Here’s a candy gift.” – the accompanying icon was also changed to a candy.

```

> ## Conclusion

Reviewer 2: I'd add some different headline, than just consluion. e.g. Conclusion: The Crucial Role of Product Localization in Global Expansion

> In conclusion, effective **product localization is vital for expanding into new markets** and connecting with diverse audiences worldwide. It goes beyond translation, encompassing **cultural understanding and language adaptation**. By considering market research, brand identity, and tone of voice, businesses can create a strong localization strategy that resonates deeply with local consumers, fostering meaningful relationships and driving sales.

Reviewer: ```suggestion
In conclusion, effective **product localization is vital for expanding into new markets** and connecting with diverse audiences worldwide. It goes beyond translation, encompassing **cultural understanding and language adaptation**. By considering market research, brand identity and tone of voice, businesses can create a strong localization strategy that resonates deeply with local consumers, fostering meaningful relationships and driving sales.

```

> Understanding cultural differences, symbolism, and imagery is crucial to **align the localized product with local preferences and avoid misinterpretations**. Addressing legal considerations such as intellectual property rights, data privacy, and product compliance is essential to safeguard brand reputation. With a clear budget, timeline, and relevant metrics, businesses can streamline the localization process and **maximize success in new markets**.

Reviewer: ```suggestion
Understanding cultural differences, symbolism and imagery is crucial to **align the localized product with local preferences and avoid misinterpretations**. Addressing legal considerations such as intellectual property rights, data privacy and product compliance is essential to safeguard brand reputation. With a clear budget, timeline and relevant metrics, businesses can streamline the localization process and **maximize success in new markets**.

```
